content/concepts/consensus-network/amendments/known-amendments.ja.md

```diff
@@ -841,13 +927,26 @@ XRP Ledgerの分散型取引所において、オファーの掛け合わせの
 バリデータは、バリデータが完全に検証されたとみなす最新の台帳のハッシュを証明する新しいオプションフィールドをバリデーションに含めることができるようにします。コンセンサスプロセスでは、この情報を使用してコンセンサスの堅牢性を高めることができます。
 
 
-## ImmediateOfferKilled
+### Hooks
+[Hooks]: #hooks
+
+| Amendment    | Hooks |
+|:-------------|:--------------------|
+| Amendment ID | ECE6819DBA5DB528F1A241695F5A9811EF99467CDE22510954FD357780BBD078 |
+| Status       | 開発中 |
+| デフォルトの投票(最新の安定版) | いいえ |
+| Amendment前の機能は廃止? | いいえ |
+
+トランザクションの前後にアカウント上で実行できる小さなコードという形式で、オンチェーンのスマートコントラクトを追加します。詳細は[Hooksドキュメント](https://xrpl-hooks.readme.io/)をご覧ください。
```

Comment:
As far as I know the Hooks documentation is only in English, so it might make sense to warn users that this link won't be available in Japanese.

```suggestion
トランザクションの前後にアカウント上で実行できる小さなコードという形式で、オンチェーンのスマートコントラクトを追加します。詳細は[Hooksドキュメント （英語のみ）](https://xrpl-hooks.readme.io/)をご覧ください。
```